Model/Model.tex

```diff
@@ -843,16 +843,21 @@ \subsubsection{Other useful global variables}
 
 \begin{itemize}
 \item
-\ct{Smalltalk} is the instance of \ct{SystemDictionary} that defines all of the globals\,---\,including \ct{Smalltalk} itself.   
-The keys to this dictionary are the symbols that name the global objects in \st code.
+\ct{Smalltalk} is the single instance of \ct{SmalltalkImage} that, inter alia, provides access to all globals\,---\,including \ct{Smalltalk} itself.
+\ct{Smalltalk globals} answers an \clsind{Environment} instance which defines all of the global bindings and provides a \clsind{Dictionary}-like interface.
```

Comment:
try to get rid of bindings. This is a low-level concept which is not relevant for readers here. They might see this more as "global variables"

Reply:
Resolved via 0d8ab089bcf48ce2ef61df8cb05c79f765dc84ec